spec/tasks/add_navigation_links_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "Navigation Links tasks", type: :task do
+  before do
+    Rake::Task.clear
+    PracticalDeveloper::Application.load_tasks
+  end
+
+  describe "#create" do
+    it "creates navigation links for new forem if nonexistent" do
+      expect { Rake::Task["navigation_links:create"].invoke }.to change(NavigationLink, :count).from(0).to(5)
```

Comment:
I generally try to make specs a bit more robust by not asserting on things that are not strictly speaking necessary. Here for example it's not so important that 0 links previously existed but that 5 new ones get created.

```suggestion
      expect { Rake::Task["navigation_links:create"].invoke }.to change(NavigationLink, :count).by(5)
```